### DISCUSSION TITLE
How to get strict verbatim

### DISCUSSION THREAD

**Discussion by ujlm** [deepgram/community#172] [General help] [answered]
Most of the time when doing transcription, we want stop words and repetitions to be directed, and Deepgram does a good job here. But some tasks require strict verbatim. What are the settings I need to apply to include words like um, uh uhu, ah, ... in the transcript?

**Comment by jjmaldonis** [comment, marked as answer]
Hey @ujlm! We are currently working on a new feature to include those words. We are calling them "filler words", so keep an eye out for anything related to that in the coming weeks. If you'd like to apply for early access, please send a message to us here: https://deepgram.com/contact-us/

**Reply by shirgoldbird** [reply]
@ujlm Just to make things even easier for you—if you'd like to be an early tester of our filler words feature, send me your Deepgram project ID directly and I can enable it for your account. My email is shir (dot) goldberg (at) deepgram (dot) com. Otherwise, we're aiming for a public release of this feature sometime in June/July.
